Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/organizer/list/TraceLocationsFragment.kt

```diff
@@ -0,0 +1,133 @@
+package de.rki.coronawarnapp.ui.eventregistration.organizer.list
+
+import android.os.Bundle
+import android.view.View
+import android.view.accessibility.AccessibilityEvent
+import android.widget.Toast
+import androidx.appcompat.widget.Toolbar
+import androidx.core.view.isGone
+import androidx.fragment.app.Fragment
+import com.google.android.material.transition.Hold
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.TraceLocationOrganizerTraceLocationsListFragmentBinding
+import de.rki.coronawarnapp.eventregistration.checkins.qrcode.TraceLocation
+import de.rki.coronawarnapp.util.DialogHelper
+import de.rki.coronawarnapp.util.di.AutoInject
+import de.rki.coronawarnapp.util.lists.decorations.TopBottomPaddingDecorator
+import de.rki.coronawarnapp.util.lists.diffutil.update
+import de.rki.coronawarnapp.util.onScroll
+import de.rki.coronawarnapp.util.ui.doNavigate
+import de.rki.coronawarnapp.util.ui.observe2
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBindingLazy
+import de.rki.coronawarnapp.util.viewmodel.CWAViewModelFactoryProvider
+import de.rki.coronawarnapp.util.viewmodel.cwaViewModels
+import javax.inject.Inject
+
+class TraceLocationsFragment : Fragment(R.layout.trace_location_organizer_trace_locations_list_fragment), AutoInject {
+
+    @Inject lateinit var viewModelFactory: CWAViewModelFactoryProvider.Factory
+    private val viewModel: TraceLocationsViewModel by cwaViewModels { viewModelFactory }
+    private val binding: TraceLocationOrganizerTraceLocationsListFragmentBinding by viewBindingLazy()
+    private val traceLocationsAdapter = TraceLocationsAdapter()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        exitTransition = Hold()
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupMenu(binding.toolbar)
+
+        binding.recyclerView.apply {
+            adapter = traceLocationsAdapter
+            addItemDecoration(TopBottomPaddingDecorator(topPadding = R.dimen.spacing_tiny))
+            onScroll {
+                onScrollChange(it)
+            }
+        }
+
+        binding.toolbar.setOnClickListener {
+            popBackStack()
+        }
+
+        viewModel.traceLocations.observe2(this) {
+            traceLocationsAdapter.update(it)
+            binding.apply {
+                recyclerView.isGone = it.isEmpty()
+                qrCodesListNoItemsGroup.isGone = it.isNotEmpty()
+            }
+        }
+
+        viewModel.showDeleteSingleDialog.observe2(this) {
+            showDeleteSingleDialog(it)
+        }
+
+        viewModel.createNewTraceLocationEvent.observe2(this) {
+            doNavigate(
+                TraceLocationsFragmentDirections
+                    .actionTraceLocationOrganizerTraceLocationsListFragmentToTraceLocationOrganizerCategoriesFragment()
+            )
```

Comment:
I think, this is what is causing the strange transition below, you need to use shared item transition as in `CheckInsFragment` 
1- transitionName of the FAB should be the same as the transitionName of the parent layout of 
`TraceLocationOrganizerCategoriesFragment`

2- Pass shared item transition bundle using NavController

https://user-images.githubusercontent.com/25054729/112508029-7a804b00-8d8f-11eb-84b5-2da681599efa.mp4